src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotifyClaimDetailsCallbackHandler.java

```diff
@@ -120,8 +115,7 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
     private SubmittedCallbackResponse buildConfirmationWithSolicitorOptions(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
 
-        if (!featureToggleService.isMultipartyEnabled()
-            || caseData.getDefendantSolicitorNotifyClaimDetailsOptions() == null) {
+        if (caseData.getDefendantSolicitorNotifyClaimDetailsOptions() == null) {
```

Comment:
You have removed this similar if condition in NotifyClaimCallbackHandler. is there a reason why we shouldn't keep it consistent here (either remove or keep at both places).

Reply:
Better to keep in both places to avoid NPE.